tasks/prime_hhg_workflow.py

```diff
@@ -131,64 +135,69 @@ def get_stored(self, object_key, object_id=None):
 
     def add_stored(self, object_key, object_data):
         """Adds the object to the main move."""
-
-        # If it's a moveTaskOrder, we only keep track of one move per workflow, so add actually replaces.
         if object_key == MOVE_TASK_ORDER:
-            self.current_move = object_data
-
-        # If it's the following object types, we add them into nested array in the MTO
-        elif object_key in [MTO_SHIPMENT, MTO_SERVICE_ITEM, PAYMENT_REQUEST]:
-            if isinstance(object_data, list):
-                self._add_nested_array_to_mto(object_key, object_data)
-            else:
-                self._add_nested_object_to_mto(object_key, object_data)
+            self.current_move_id = object_data["id"]
+
+        # # If it's a moveTaskOrder, we only keep track of one move per workflow, so add actually replaces.
```

Comment:
@NamibiaTorres you might have considered this already but I'm curious about keeping the old code in this file? Since we have a version history I think it would be fine to get rid of it

Reply:
Thanks I was really unsure about keeping it commented out. I guess I can jsut add a note to look at this PR if we decide to bring it back and not use the move_id.